test/apps/viewer/src/app.js

```diff
@@ -103,23 +117,45 @@ class Example extends PureComponent {
       })
       .on('error', console.error)
       .connect();
+
+    // Monitor the log
```

Comment:
Updated the example app to reflect how the XVIZWorkersMonitor is now decoupled from the perf view components and its metrics passed as props to those view components.